.github/workflows/run-k8s-tests-on-aks.yaml

```diff
@@ -45,9 +45,18 @@ jobs:
           - host_os: cbl-mariner
             vmm: clh
             instance-type: small
+            use-policy: true
+            genpolicy-pull-method: oci
```

Comment:
@danmihai1 Does `oci` make sense? I remember you mentioned `oci-distribution` before.